LayoutTests/http/tests/cookies/third-party-cookie-relaxing.https.html

```diff
@@ -1,6 +1,6 @@
 <html>
 <head>
-<script src="resources/resetCookies.js"></script>
+<script src="resources/resetCookiesSecure.js"></script>
```

Comment:
I am not sure about the other tests but I don't think this test needs https.html -- we need https.html when the test itself needs to be hosted on security origin, not subresources. 
I know this test as I have an ongoing PR https://github.com/WebKit/WebKit/pull/28581 for it. Maybe you could merge that change in this patch, and see whether the related tests need the renaming.

Reply:
The tests that only set cookies from an iframe don't need the mainframe to also be under https, I can change those. The tests that set cookies from the main resource do need the renaming.

I'll undo the renaming for:
third-party-cookie-relaxing.html
access-control-preflight-credential-async.html
access-control-preflight-credential-sync.html